### ISSUE TITLE
Tracker not working as of v2.14

### ISSUE THREAD

**Issue by tradenet** [mautic/mautic-helper-chrome-extension#7]
As reported at Google Web Store as well. 
When the helper is checked, nothing is inserted in mail body.

The email has no tracking code when view from recipient end.

Running Mautic v2.14.0

**Comment by socialnicheguru** [comment]
i am getting this too

**Comment by tifaucz** [comment]
Same bug here

**Comment by pravinuttarwar** [comment]
Any luck here? Not working with new gmail interface

**Comment by zietbukuel** [comment]
Same here, any news?

**Comment by bgronek** [comment]
1+ years later...is this extension still supported?  It sure does seem useful!

**Comment by tradenet** [comment]
> 1+ years later...is this extension still supported? It sure does seem useful!

Exactly. Now that there's a "new team" on the block I hope it will be.

**Comment by mfiriondo** [comment]
Still not working.

**Comment by RCheesley** [comment]
Hi folks,

If someone has the expertise to address the issues we are absolutely open to PR's to fix this.

At present we do not have the skillset needed in the product team, so it would need a developer to pick it up and update it, or a sponsor to find someone to work on this and contribute it back to the community.
